packages/main/src/DateTimePicker.ts

```diff
@@ -210,9 +204,8 @@ class DateTimePicker extends DatePicker {
 	 * @public
 	 */
 	async openPicker(): Promise<void> {
-		await super.openPicker();
-		this._currentTimeSlider = "hours";
 		this._previewValues.timeSelectionValue = this.value || this.getFormat().format(new Date());
+		await super.openPicker();
```

Comment:
What's the idea behind changing the place where super.openPicker is called?

Reply:
There is possible timing issue, if there is no order change - it just doesn't work as expected.

Reply:
Could you please share what's the problem with previous order because I'm playing with both and I'm not able to find a difference.

Also in my understanding `super` should be executed before other logic since with this override we want to override the default behaviour or to enhance it.

Reply:
After you found the root of the problem, "reordering" is fixed.